app/javascript/components/public_health/PublicHealthHeader.js

```diff
@@ -127,7 +127,7 @@ class PublicHealthHeader extends React.Component {
           {this.state.importType === 'saf' && (
             <div className="mb-3">
               <a href="https://github.com/SaraAlert/SaraAlert/blob/master/public/Sara%20Alert%20Import%20Format.xlsx?raw=true">Download formatting guidance</a>{' '}
-              (Updated 2/9/2021)
+              (Updated 2/24/2021)
```

Comment:
Is anyone else running into this problem?
1. Check out this branch and do a git pull or hard reset to update to the latest version of the remote branch
2. Open the import file and close it without modifying or saving it
3. `git status` shows that the file was modified
4. `git reset --hard HEAD` shows this message: `Encountered 1 file(s) that should have been pointers, but weren't: public/Sara Alert Import Format.xlsx`

I'm wondering if there's anything weird going on with git lfs here, I haven't seen this happen on other branches with modifications to this file so I'm wondering if it's specific to the changes on this one. @sgober you have git lfs installed and everything right?

Reply:
also, in case anyone is running into this problem, you can do a hard lfs reset by running:
```
git lfs uninstall
git reset --hard
git lfs install & git lfs pull
```

Reply:
aha that should have done the trick - lmk if you're still having this issue

Reply:
looks like it's fixed!